### ISSUE TITLE
Change padding method

### ISSUE THREAD

**Issue by fangwei123456** [fangwei123456/spikingjelly#77]
https://github.com/fangwei123456/spikingjelly/blob/e2aacded5df95b2a5f449090b34593af3845cd35/spikingjelly/datasets/__init__.py#L164

When we use fixed duration=1e6 in DVS Gesture, we will get max frames number = 19, and all frames will be padded with frames number = 19. However, the frames number of most frames are much smaller than 19. But the current padding method does not provide pack method. I will change the API by `torch.nn.utils.rnn.pad_sequence` and `torch.nn.utils.rnn.pack_padded_sequence`.
